app/models/reservation_scopes.rb

```diff
@@ -50,6 +50,9 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :checkoutable, lambda {
         where('start_date <= ?', Time.zone.today).reserved
       }
+      scope :future, lambda {
```

Comment:
~~This would be replaced with the~~ We would add a `pending` scope that would look for untouched reservations reserved in the range from `Time.zone.today` until `Time.zone.today + 8.days`.